code/web/src/modules/survey/StyleSurvey.js

```diff
@@ -35,7 +55,8 @@ class StyleSurvey extends Component {
   }
 
   submitSurvey = (e) => {
-    console.log('giving to backend!')
+    e.preventDefault()
+    this.getStyleTally()
```

Comment:
I will add the mutations as helper functions through here

Reply:
Yep! That's where intended on that functionality living. Looks good to merge!